src/focus_manager/txm_platform.js

```diff
@@ -70,8 +70,10 @@ export class TXMPlatform {
         this.isPS4 = false;
         this.isPS5 = false;
 
-        this.isFireTV = false;
+        this.isAndroid = false; // true for both android mobile and android tv
```

Comment:
Suggest making this derived off of isAndroidMobile and isAndroidTV instead.  I don't see other cases, but we could add it to the above as well if there are any.

Reply:
well, I set it to true for those cases only
otherwise I could have it as a computable property
I find myself preferring the simple property approach to allow test drivers to play with mock data

Reply:
I guess I find it simpler not to worry about setting it manually per platform, granted these two might be the only ones? But if you think it is useful for testing as standalone, then thats fine.

Reply:
i don't actually seem to need to override it at the moment, i can make it computed